package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
 	"name": "vscode-buf",
-	"version": "0.1.0",
+	"version": "0.1.1",
```

Comment:
why is this 0.1.1 when the new version is 0.1.2

Reply:
I asked this last time and the answer was "npm"

Reply:
kk